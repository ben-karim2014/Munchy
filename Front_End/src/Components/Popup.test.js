import React from 'react';
import renderer from 'react-test-renderer';

import PopupPrompt from './Popup';

it('renders correctly when adding ingredients', () => {
    const isOpenItem = true;
    const tree = renderer.create(
    <PopupPrompt isOpen={isOpenItem}>
        Please Enter the Ingredient
    </PopupPrompt>).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly when adding recipe URL', () => {
    const isOpenItem = true;
    const recipePromptMessage = "Please Enter the Recipe URL";
    const tree = renderer.create(
    <PopupPrompt isOpen={isOpenItem}>
        {recipePromptMessage}
    </PopupPrompt>).toJSON();
    expect(tree).toMatchSnapshot();
});

