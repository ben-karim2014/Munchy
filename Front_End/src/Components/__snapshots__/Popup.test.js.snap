// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when adding ingredients 1`] = `
<div>
  <strong>
    <div
      className="modal-container modal-center round-border"
    >
      <div
        className="modal-div-right"
      >
         
        <button
          aria-label={null}
          className="modal-button btn btn-secondary"
          onClick={[Function]}
        >
          x
        </button>
      </div>
      <div>
        <p>
          Please Enter the Ingredient
        </p>
        <input
          className="modal-textarea"
          id="ingredients"
          maxLength="100"
          onChange={[Function]}
        />
      </div>
      <div
        className="modal-div-right"
      >
        <button
          aria-label={null}
          className="modal-common-button btn btn-secondary"
          id="add-button"
          onClick={[Function]}
        >
          Add
        </button>
      </div>
    </div>
  </strong>
</div>
`;

exports[`renders correctly when adding recipe URL 1`] = `
<div>
  <strong>
    <div
      className="modal-container modal-center round-border"
    >
      <div
        className="modal-div-right"
      >
         
        <button
          aria-label={null}
          className="modal-button btn btn-secondary"
          onClick={[Function]}
        >
          x
        </button>
      </div>
      <div>
        <p>
          Please Enter the Recipe URL
        </p>
        <input
          className="modal-textarea"
          id="ingredients"
          maxLength="100"
          onChange={[Function]}
        />
      </div>
      <div
        className="modal-div-right"
      >
        <button
          aria-label={null}
          className="modal-common-button btn btn-secondary"
          id="add-button"
          onClick={[Function]}
        >
          Add
        </button>
      </div>
    </div>
  </strong>
</div>
`;
